<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogWindow</class>
 <widget class="QDialog" name="CatalogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Каталог автомобилей</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 240, 239);
font: 600 10pt &quot;Montserrat&quot;;
color: rgb(55, 51, 53);</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget
{
	background-color: rgb(244, 240, 239);
	border: none;
	border-radius: 15px;
	color: rgb(55, 51, 53);
	font: 9pt &quot;Montserrat&quot;;
	outline: 0px;
}
QListWidget::item
{
	outline: none;
	padding: 2px;
	margin-right: 10px;
}
QScrollBar
{
	background-color: rgb(244, 240, 239);
	border: 2px solid rgb(244, 240, 239);
	width: 15px;
}
QScrollBar::handle:vertical
{
	border: 2px solid blue;
	background-color: rgb(244, 240, 239);
	border-radius: 2px;
	width: 5px;
}
 QScrollBar::add-line:vertical {
     border: 2px solid rgb(55, 51, 53);
     background:  rgb(244, 240, 239);
     height: 20px;
 }</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #9AB6AC;
}
QPushButton{
	color: rgb(55, 51, 53);
	font: 600 10pt &quot;Montserrat&quot;;
	border-radius: 10px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(200, 215, 210);
	padding: 5px;
}

</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="topLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 11pt &quot;Montserrat&quot;;</string>
   </property>
   <property name="text">
    <string>Каталог автомобилей</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>177</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #CC3329;
}
QPushButton{
	color: rgb(244, 240, 239);
	font: 600 10pt &quot;Montserrat&quot;;
	border-radius: 12px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(215, 67, 57);
	padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #CC3329;
}
QPushButton{
	color: rgb(244, 240, 239);
	font: 600 10pt &quot;Montserrat&quot;;
	border-radius: 12px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(215, 67, 57);
	padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>119</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 10pt &quot;Montserrat&quot;;
</string>
   </property>
   <property name="text">
    <string>       Действия
с автомобилями</string>
   </property>
  </widget>
  <widget class="QLabel" name="compareLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;</string>
   </property>
   <property name="text">
    <string>Чтобы сравнить два
автомобиля, выберите
первый, нажмите кнопку,
затем выберите второй
и нажмите снова.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comparePushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #9AB6AC;
}
QPushButton{
	color: rgb(55, 51, 53);
	font: 600 10pt &quot;Montserrat&quot;;
	border-radius: 10px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(200, 215, 210);
	padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLabel" name="filterLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отобразить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="filterComboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox 
{
	color: rgb(30, 36, 39);
	font: 600 10pt &quot;Montserrat&quot;;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(200, 215, 210);
	padding: 5px;
	outline: none;
}
QComboBox::drop-down
{
	border:none;
}
QComboBox::down-arrow
{
	image: url(:/rcs/free-icon-down-arrow-2985150.png);
	wigth: 30px;
	height:30px;
	margin-right: 5px;
}
QComboBox QAbstractItemView::item:selected
{
	background-color:rgb(244, 240, 239);
	border-left: 2px solid #CC3329;
	outline:none;
}
QComboBox QAbstractItemView::item
{
	outline: none;
	border-left: 2px solid rgb(244, 240, 239);
}
QComboBox QAbstractItemView
{
	outline: none;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Все автомобили</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Только электрические</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Только с ДВС</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Только гибридные</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
